docs/source/version_compatibility.rst

```diff
@@ -0,0 +1,34 @@
+**************
+Sabre Versions
+**************
+
+The following table explains what versions of the Sawtooth Sabre SDK are
+compatible with different versions of the Sawtooth Sabre transaction
+processors.
+
+The Sabre SDK versions are the Crate versions of the Rust library that should
+be set in the Cargo.toml file and the Sabre transaction processor versions are
+the versions used as part of the transaction processor’s registration.
+
+The docker tag is the tag that should be used for the
+hyperledger/sawtooth-sabre-tp image if including in a docker-compose yaml file.
+
++-------------------+------------+------------+--------------------------------+
+| Sabre Transaction | Sabre SDK  | Docker Tag | Description                    |
+| Processor         | (Cargo     |            |                                |
+| (Family Version)  | Version)   |            |                                |
++===================+============+============+================================+
+| 0.0               | 0.1        | 0.1        | Compatible with the 0.2 version|
+|                   |            |            | of the Sawtooth Rust SDK.      |
++-------------------+------------+------------+--------------------------------+
+| 0.2               | 0.2        | 0.2        | Compatible with the 0.3 version|
+|                   |            |            | of the Sawtooth Rust SDK, which|
```

Comment:
Also, it would be nice to use bullets for the two items, if time allows. (If the formatting gets tricky, feel free to leave this change for later.)

For example:

* Transaction context is a trait

* API has new get_state_entry to get one entry and get_state_entries to get multiple entries (plus corresponding functions for set and delete)